pkgs/top-level/all-packages.nix

```diff
@@ -24575,7 +24575,6 @@ in
   };
 
   wlroots_0_12 = callPackage ../development/libraries/wlroots/0.12.nix {};
-  wlroots_0_13 = callPackage ../development/libraries/wlroots/0.13.nix {};
```

Comment:
Maybe an alias should have been added for this.

Reply:
We've added it just a few days ago to ease the transition to wlroots 0.14 so no one should depend on it (it was basically a deprecated alias from the beginning but AFAIK we don't have a way to mark it as such if packages in Nixpkgs still depend on it). Anyway, in this case it should be fine.